snap-geometry-edits/README.md

```diff
@@ -12,7 +12,7 @@ A field worker can create new features by editing and snapping the vertices of a
 
 To create a geometry, press the create button to choose the geometry type you want to create (i.e. points, multipoints, polyline, or polygon) and interactively tap and drag on the map view to create the geometry.
 
-To configure snapping, press the snap settings button to enable or disable snapping and choose which snap sources to snap to.
+Snap settings can be configured by enabling and disabling snapping, feature snapping, geometry guides and snap sources.
```

Comment:
FYI I believe it's standard to use the oxford comma in lists like this that end with a conjunction (i.e. use a comma before the "and"):

```suggestion
Snap settings can be configured by enabling and disabling snapping, feature snapping, geometry guides, and snap sources.
```